ivy/functional/frontends/tensorflow/math.py

```diff
@@ -549,6 +549,19 @@ def tanh(x, name=None):
 def rsqrt(x, name=None):
     return ivy.reciprocal(ivy.sqrt(x))
 
+@to_ivy_arrays_and_back
+def segment_sum(data, segment_ids, name= "segment_sum"):
+    data = ivy.array(data)
```

Comment:
You don't need to change the arrays to ivy arrays. The `to_ivy_arrays_and_back` decorator handles that